conda_smithy/cli.py

```diff
@@ -289,9 +289,10 @@ class Rerender(Regenerate):
         args = parser.parse_args()
 
     # Check conda version for compatibility
-    CONDA_VERSION_MAX = '4.6'
+    CONDA_VERSION_MAX = '4.7'
     if LooseVersion(conda.__version__) >= LooseVersion(CONDA_VERSION_MAX):
-        print('You appear to be using conda {}, but conda-smithy {} is\ncurrently only compatible with conda versions < {}.'.format(
+        print('You appear to be using conda {}, but conda-smithy {} '
+              'is\ncurrently only compatible with conda versions < {}.'.format(
```

Comment:
nit: Would be nice to terminate the previous line with the `\n` so it doesn't get lost in the text below.